src/app/services/expense.service.ts

```diff
@@ -31,59 +28,31 @@ export class ExpenseService {
   async captureExpenseReceipt() {
     const options: CameraOptions = {
       quality: 100,
-      destinationType: this.camera.DestinationType.FILE_URI,
-      encodingType: this.camera.EncodingType.JPEG,
-      mediaType: this.camera.MediaType.PICTURE
-    }
-
-    const imageData = await this.camera.getPicture(options);
-    const resolvedImg = this.webview.convertFileSrc(imageData);
-    return {
-      sanitizedReceiptImage: this.sanitizer.bypassSecurityTrustUrl(resolvedImg),
-      originalImage: imageData
+      resultType: CameraResultType.Base64
```

Comment:
wiped out my beautiful CRUD stuff! haha.  Can we merge this PR after we get all platforms working? Don't want to break ios/android... I think it wouldn't be too difficult, actually. I implemented almost the same functionality in the [First App tutorial](https://ionicframework.com/docs/angular/your-first-app).

Reply:
Yeah, it's not really meant to be merged in right now.  Want me to change to WIP?